patchmatch/patch_match.py

```diff
@@ -156,38 +157,18 @@ def download_url_to_file(url, dst, hash_prefix=None, progress=True):
             if lib_name.startswith("libpatchmatch_"):
                 pypatchmatch_lib = lib_name
 
-    # Compile if we didn't find a platform-compatible version (and it's not compiled already)
     if pypatchmatch_lib is None:
         pypatchmatch_lib = "libpatchmatch.so"
```

Comment:
when I change this to libpatchmatch.dylib, it seems to be working properly on my Mac

Reply:
I've added a check to use than name when platform is `darwin`.

It's suspicious to me to have to hardcode these extensions here rather than using a more standardized interface that already knows these platform-specific conventions, but I didn't find one of those.

Reply:
I was playing around with Windows to see if I can get it working there as well, but I only have access to a VM which runs on ARM, which is why I think I am failing ...

What's nagging me about the `dylib`: When I build the main branch, it is generating a `libpatchmatch.so` instead. So maybe there is a attribute in meson.build to change that behaviour 🤔